packages/docs/pages/MultiSelect/MultiSelectPage.tsx

```diff
@@ -0,0 +1,286 @@
+                  { value: 'ch', content: 'Chile' },
+                  { value: 'bo', content: 'Bolivia' },
+                  { value: 'jp', content: 'Japan' },
+                  { value: 'cn', content: 'China' },
+                  { value: 'sk', content: 'South Korea' },
+                  { value: 'au', content: 'Australia' },
+                  { value: 'ug', content: 'Uganda' },
+                ]}
+                placeholder={'Choose country'}
+                placement={'bottom-start'}
+                required
+                value={value}
+              />
+            </FormGroup>
+          </Form>
+        );
+      }}
+      {/* jsx-to-string:end */}
+    </CodePreview>
+
+    <H1>API</H1>
+    <MultiSelectPropTable />
+    <SelectOptionPropTable />
+    <SelectActionPropTable />
+
+    <H1>Position</H1>
+
+    <Text>
+      MultiSelect can be anchored in different directions with the <Code primary>position</Code> property. It will
+      automatically find a position if there's not enough space in the chosen direction.
+    </Text>
+
+    <CodePreview>
+      {/* jsx-to-string:start */}
+      <Grid gridColumns="repeat(4, 1fr)">
+        <MultiSelect
+          label="Select"
+          onOptionsChange={() => null}
+          options={[
+            { value: 1, content: 'Option' },
+            { value: 2, content: 'Option' },
+            { value: 3, content: 'Option' },
+            { value: 4, content: 'Option' },
+          ]}
+          placeholder="Choose from above"
+          placement="top"
+          required
+        />
+        <MultiSelect
+          label="Select"
+          onOptionsChange={() => null}
+          options={[
+            { value: 1, content: 'Option' },
+            { value: 2, content: 'Option' },
+            { value: 3, content: 'Option' },
+            { value: 4, content: 'Option' },
+          ]}
+          placeholder="Choose from below"
+          placement="bottom-start"
+          required
+        />
+        <MultiSelect
+          label="Select"
+          onOptionsChange={() => null}
+          options={[
+            { value: 1, content: 'Option' },
+            { value: 2, content: 'Option' },
+            { value: 3, content: 'Option' },
+            { value: 4, content: 'Option' },
+          ]}
+          placeholder="Choose from the right"
+          placement="right-start"
+          required
+        />
+        <MultiSelect
+          label="Select"
+          onOptionsChange={() => null}
+          options={[
+            { value: 1, content: 'Option' },
+            { value: 2, content: 'Option' },
+            { value: 3, content: 'Option' },
+            { value: 4, content: 'Option' },
+          ]}
+          placeholder="Choose from the left"
+          placement="left-end"
+          required
+        />
+      </Grid>
+      {/* jsx-to-string:end */}
+    </CodePreview>
+
+    <H1>Max Height</H1>
```

Comment:
Example titles should be `H2`'s. See `PAGES.md`.
```suggestion
    <H1>Examples</H1>
    
    <H2>Max Height</H2>
```